frame/contracts/src/exec.rs

```diff
@@ -1227,6 +1236,15 @@ where
 		});
 		Ok(())
 	}
+
+	fn reentrant_count(&mut self) -> u32 {
+		let id: AccountIdOf<Self::T> = self.top_frame().account_id.clone();
```

Comment:
Why do you need to clone it here? Maybe you can account by ref?

Reply:
updated